README.md

```diff
@@ -59,6 +60,13 @@ services.AddSitemaps(x =>
 }, p => p.RequireRole(Roles.Administrators));
 ```
 
+In order to augment Urls you must remove the default IUriAugmenterService and implement your own:
+```csharp
+// Swap out the NullUriAugmenterService so we can augment Urls within the Sitemap.
+services.RemoveAll<IUriAugmenterService>();

```

Comment:
Instead of documenting how to add/remove augmenter service, provide configuration options that allow setting AugmenterService in ServiceCollectionExtensions.